src/CloudWatchLogs/CloudWatchLogsClient.php

```diff
@@ -155,4 +156,61 @@
  * @method \Aws\Result updateLogAnomalyDetector(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateLogAnomalyDetectorAsync(array $args = [])
  */
-class CloudWatchLogsClient extends AwsClient {}
+class CloudWatchLogsClient extends AwsClient {
+    static $streamingCommands = [
+        'StartLiveTail' => true
+    ];
+
+    public function __construct(array $args)
+    {
+        parent::__construct($args);
+        $this->addStreamingFlagMiddleware();
+    }
+
+    private function addStreamingFlagMiddleware()
+    {
+        $this->getHandlerList()
+            -> appendInit(
+                $this->getStreamingFlagMiddleware(),
+                'streaming-flag-middleware'
+            );
+    }
+
+    private function getStreamingFlagMiddleware(): callable
+    {
+        return function (callable $handler) {
+            return function (CommandInterface $command, $request = null) use ($handler) {
+                if (!empty(self::$streamingCommands[$command->getName()])) {
+                    $command['@http']['stream'] = true;
+                }
+
+                return $handler($command, $request);
+            };
+        };
+    }
+
+    /**
+     * Helper method for 'startLiveTail' operation that checks for results.
+     *
+     * Initiates 'startLiveTail' operation with given arguments, and continuously
+     * checks response stream for session updates or results, yielding each
+     * stream chunk when results are not empty.
+     *
+     * @param array $args Command arguments.
+     *
+     * @return \Generator Yields session update or result stream chunks.
+     */
+    public function startLiveTailCheckingForResults(array $args): \Generator
```

Comment:
I think the backslash in Generator is throwing off the type removal script.  If I'm not mistaken, it's looking only for alphabetical characters.  This will need to be updated here and in our release system.

Reply:
I removed.